doc/ctc-meetings/2016-01-20.md

```diff
@@ -0,0 +1,337 @@
+* ArrayBuffer.isView() and buffer.buffer property [#4420](https://github.com/nodejs/node/issues/4420)
+* doc: add Docs working group [#4244](https://github.com/nodejs/node/pull/4244)
+* Seek legal advice on LICENSE and copyright blocks in code [#3979](https://github.com/nodejs/node/issues/3979)
+* fs: optimize realpath using uv_fs_realpath() [#3594](https://github.com/nodejs/node/pull/3594)
+
+## Standup
+
+* James Snell (CTC): Working on the Buffer issue, work on HTTP module, reviewing issues and PRs, attempting to help on 4.2.5,
+* Trevor Norris (CTC): AsyncWrap related things
+* Colin Ihrig (CTC): Reviewing issues and PRs. Reverted a commit
+* Brian White (CTC): Not much, commenting on PRs and issues.
+* Alexis Campailla (CTC): Fixing Windows issues on libuv and npm
+* Bert Belder (CTC): _[CD - missed what you were looking into due to cross talk]_
+* Mikeal Rogers (observer): Various projects coming into the Foundation
+* Jeremiah Senkpiel (CTC): Onboarding new release team members, Working on Timers Refactor more, trying to make sure Chakra and Buffer discussions remain productive
+* Rod Vagg (CTC): Working with new contributors, lots of meetings
+* Chris Dickinson (CTC): Docs WG stuff
+* Shigeki Ohtsu (CTC):
+* Steven Loomis (observer):
+* Ben Noordhuis (CTC): Been sick, still catching up.
+* Rich Trott (observer): Making tests more reliable, spinning up Testing WG
+* Nikita Skovoroda “Chalker” (observer): mostly buffer, also the api usage greps are now sorted based on downloads count
+* Ali Sheikh (observer): sampling heap profiler in V8
+* Domenic (observer): modules and zones
+* Evan Lucas (observer): Working on getting v5.5.0 Release out and looking into v8 extras for some of the builtins.
+
+## Review of last meeting
+
+* Nominating new Release team members [#4319](https://github.com/nodejs/node/issues/4319)
+* repl: Reverses order of .node_repl_history [#4313](https://github.com/nodejs/node/pull/4313)
+* doc: add Docs working group [#4244](https://github.com/nodejs/node/pull/4244)
+* Seek legal advice on LICENSE and copyright blocks in code [#3979](https://github.com/nodejs/node/issues/3979)
+* Potential Licensing issues with npm [#3959](https://github.com/nodejs/node/issues/3959)
+* Joyent Copyright still in header of many files [#3926](https://github.com/nodejs/node/issues/3926)
+* gripe: deprecating fs.exists/existsSync [#1592](https://github.com/nodejs/node/issues/1592)
+
+## Minutes
+
+### Enable Node.js to run with Microsoft's ChakraCore engine [#4765](https://github.com/nodejs/node/pull/4765)
+
+Put on agenda by Alexis
+
+Alexis: MS submitted a PR for supporting Chakracore, change in node is fairly small — the shim is in the deps folder. A few CTC members have expressed interest in landing. I was wondering if anyone had opinions about how to proceed with this?
+
+Domenic: I’m curious about what level of support is implied. Landing it reduces the requirement of maintaining a separate fork, but
```

Comment:
Yeah, there are a few in this one unfortunately. I think the pace of discussion was a bit too fast in this meeting for good minuting. We had a lot to cover and were all conscious of time, plus there were some pretty strong opinions to be had!